help/destinations/catalog/advertising/acxiom-audience-distribution.md

```diff
@@ -0,0 +1,143 @@
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the [!DNL Acxiom Audience Distribution] destination, here is a sample use case that [!DNL Adobe Experience Platform] customers can solve by using this connector.
+
+### Send audiences from Experience Platform to your Acxiom account {#send-audiences}
+
+Use this destination connector if you are a marketing professional who wants to send audiences from [!DNL Experience Platform] to your [!DNL Acxiom] account, for cross-channel acquisition.
+
+For example, the Marketing Operations department at a global financial services brand uses the [!DNL Acxiom Audience Distribution] destination to send audiences from [!DNL Experience Platform] to [!DNL Acxiom] for cross-channel acquisition.
+
+
+While the segmentation runs, you must set up the connection to a specific destination (e.g., [!DNL LG Ads], [!DNL Spectrum], [!DNL Altrice], etc.).
+
+Once the connection is set up and the audience is created, you can schedule the data flow within the destination card.
+
+## Prerequisites {#prerequisites}
+- **Confirm Terms of Use:** Before you can configure a new [!DNL Acxiom Audience Distribution] destination, you must read and sign [!DNL Acxiom’s] Terms of Use Agreement. You will receive the link to the agreement once your executed sales order is complete. Until you sign the agreement, you will not see the [!DNL Acxiom Audience Distribution] destination card in the Experience Platform destination catalog. After you accept and sign the agreement, [!DNL Adobe] will complete your onboarding process and you will see the [!DNL Acxiom Audience Distribution] destination card.
+- **Know your Adobe organization ID:** Your [!DNL Adobe] organization ID is needed to complete your Terms of User Agreement. See [!DNL Adobe's] *Organizations in Experience Cloud* topic for details on how to [view your organization ID](https://experienceleague.adobe.com/en/docs/core-services/interface/administration/organizations#concept_EA8AEE5B02CF46ACBDAD6A8508646255).
+
+
+## Supported Destinations {#supported-destinations}
+
+[!DNL Acxiom’s Audience Distribution] destination currently supports audience activation to the following platforms.<br> 
+	
+- [!DNL Altice]
+- [!DNL Ampersand]
+- [!DNL Comcast]
+- [!DNL Cox]
+- [[!DNL LG Ads]](#lg-ads)
+- [!DNL Spectrum]
+- [!DNL Viant]
+
+
+## Connect to a new destination {#connect}
+Authentication to [!DNL Acxiom's Audience Distribution] destination is automatically handled behind the scenes for your convenience.
+
+
+## Destinations specific settings {#destination-specific-settings}
+Some [!DNL Acxiom Audience Distribution] destinations require additional information. The sections below provide detailed guidance on how to setup the destinations that have specific configuration options.

```

Comment:
Some [!DNL Acxiom Audience Distribution] destinations require additional information. The sections below provide detailed guidance on how to configure these options.

Reply:
Done. Thank you!